### DISCUSSION TITLE
Learn how to create next generation applications with vector databases and machine learning!

### DISCUSSION THREAD

**Discussion by cassiebreviu** [community/community#64130] [Programming Help]
Vector databases are an essential component of modern machine learning techniques. How do you vectorize data and how does that work with machine learning? Essentially, they can store multiple types of data in a way that computers can process more efficiently, using numerical vectors that capture the meaning of the data. Retrieval Augmented Generation (RAG) is one such technique that uses vector databases to enhance the reliability and ground data for generative AI models to improve the accuracy of the generated information. These capabilities can be leveraged through cloud services that provides vector database capabilities, enabling semantic search, recommendation engines, and search rich media. To learn more about this, check out this 3 video series from The AI Show on vectors and vector search leveraging the power of OpenAI Models!

Part 1: [Igniting Developer Innovation with Vector Search](https://youtu.be/otlYeBH9U6E)
Part 2 and 3: [Combining the power of vector search with Azure OpenAI then revolutionize image search with vectors!](https://www.youtube.com/live/eJTbNYFYODY?feature=share&t=102)

**Comment by hith3sh** [comment]
Thank you for the two videos you mentioned above, watched it & gained some real info about how vector databases improve the future of ML.
I have a small question btw, can you show me some real examples of tasks where RAG using vector databases has shown significant improvements over traditional generative AI models ?
thanks ;)
